process explorer/backend/ProcessExplorer/Processes/ProcessMonitor.cs

```diff
@@ -17,6 +17,11 @@ public class ProcessMonitor : IProcessMonitor
         /// </summary>
         public static int ComposePID { get; set; }
 
+        /// <summary>
+        /// Delay time for keeping the process in the list after it is terminated.
+        /// </summary>
+        public static int DelayTime { get; set; } = 60000;
```

Comment:
I would add the unit in the comment to make it clear that this number is interpreted as **milliseconds**, not seconds or minutes, etc.
That brings up the other question: Do we want this to be controlled with millisecond precision? I think **seconds** would be a better unit.